src/openapi/streaming/streaming.spec.js

```diff
@@ -71,6 +66,7 @@ describe('openapi Streaming', () => {
         fetchMock = mockFetch();
 
         installClock();
+        mockMathRandom();

```

Comment:
this fixes max callstack exceeded error on local
the reason was Math.random is being used to get pivot index for quick sort in source map node modules
and since it always returns 0, quicksort runs with worst complexity